src/slack/dispatch.js

```diff
@@ -5,6 +5,11 @@ const { getDisplayNumber } = require("./phone-number-utils");
 
 const channel = config.SLACK_CHANNEL_ID;
 
+const pluralize = (num, str) => {
```

Comment:
In #42, all message formatting helpers have been moved to `src/slack/message/index.js`. This one should likely move there as well.